apimartifacts/apis/swagger-petstore/specification.yaml

```diff
@@ -10,8 +10,8 @@ info:
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   version: '1.0'
 servers:
-  - url: http://klndcklnsvlde-apim-dev.azure-api.net
-  - url: https://klndcklnsvlde-apim-dev.azure-api.net
+  - url: http://apim-pktmtbrc-dev.azure-api.net
```

Comment:
Should this go in some config file too? Happy for this PR to go in and we solve with another task (feel free to resolve)

Reply:
Absolutely, I've added an item to handle all hardcoded values. There are a few resource IDs and urls that should be taken from Terraform outputs, etc and written to these files rather than hardcoded. I've added item #22 to reflect this.